mrmustard/__init__.py

```diff
@@ -41,6 +40,7 @@ def __init__(self):
         self.AUTOCUTOFF_MAX_CUTOFF = 100
         self.AUTOCUTOFF_MIN_CUTOFF = 1
         # using cutoff=5 for each mode when determining if two transformations in fock repr are equal
+        self.CIRCUIT_DECIMALS = 3
```

Comment:
reverse ordering of this two lines to keep the comment above the respective line
```suggestion
        self.CIRCUIT_DECIMALS = 3
        # using cutoff=5 for each mode when determining if two transformations in fock repr are equal
```